app/main/controllers/template/rtmedia-actions.php

```diff
@@ -124,13 +131,15 @@ function rtmedia_add_album_selection_field( $media_type ) {
 			<?php
 			if ( isset( $rtmedia_query->query['context'] ) && 'group' === $rtmedia_query->query['context'] ) {
 				// show group album list.
-				$album_list = rtmedia_group_album_list( $selected_album_id = $curr_album_id );
+				$selected_album_id = $curr_album_id;
```

Comment:
Do we need this assignment? Could we directly use `$curr_album_id` and remove usage of `$selected_album_id`?

Reply:
yes. Doing it.